pages/contact.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:200,300,300i,400,700&display=swap" rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="../stylesheets/style.css">
+    <link rel="stylesheet" href="../stylesheets/contact.css">
+    <link rel="stylesheet" href="../stylesheets/animate_contact.css">
+
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <!-- Header -->
+    <header class="header">
+        <div class="container">
+            <nav class="nav" id="nav">
+                <a class="nav__link" href="../index.html">home</a>
+                <a class="nav__link" href="portfolio.html">projects</a>
+                <a class="nav__link" href="about.html">about me</a>
+                <a class="nav__link" href="blog.html">blog</a>
+                <a class="nav__link" href="contact.html">contact</a>
+            </nav>
+        </div>
+    </header> 
+
+    <!-- Contacts -->
+    <div class="contacts">
+        <div class="container">
+            <input type="button" value="Press this button to send me an email" onclick="window.location.href='mailto:ninochca@gmail.com'">
```

Comment:
This can be achieved more cleanly by using an `a` and `button` tag:

```suggestion
            <a href="mailto:ninochca@gmail.com">
                <button>Press this button to send me an email</button>
            </a>
```